packages/venia-ui/lib/components/ProductOptions/swatch.js

```diff
@@ -61,10 +61,11 @@ const Swatch = props => {
 
     return (
         <button
-            onClick={handleClick}
-            title={label}
             className={className}
+            onClick={handleClick}
             style={finalStyle}
+            title={label}
+            type="button"
```

Comment:
Swatch and tile were missing type="button" prop, and were triggering form submission on selection.

Reply:
I was wondering why you re-ordered the props, now I get it Alphabetical order.

Is that OCD or our standard? I ask because, I have OCD too 😆